rerun_js/web-viewer-react/README.md

```diff
@@ -21,7 +21,7 @@ npm i @rerun-io/web-viewer-react
 ```
 
 ℹ️ Note:
-The package version is equal to the supported Rerun SDK version, and [RRD files are not yet stable across different versions](https://github.com/rerun-io/rerun/issues/6410).
+The package version is equal to the supported Rerun SDK version, and RRD files are only compatible across different (adjacent) versions since 0.23 (see [here](https://github.com/rerun-io/rerun/issues/6410) and [here](https://rerun.io/blog/release-0.23)).
```

Comment:
let's not reference closed tickets. Actually I'm inclined to leave out the rrd compat story here altogether. If you're here you already produced some RRD so chanced you stumbled of the other notes about it.

```suggestion
The package version is equal to the supported Rerun SDK version.
```